### PR TITLE
Harbor upgrade and security patch

### PR DESCRIPTION
**Pull request by vincentni**
*Issue #, if available:*

*Description of changes:*
upgrade Harbor to v2.5.0
upgrade Harbor Helm to v1.9.0

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Review by jaxesn — CHANGES_REQUESTED** [review]
Could you add an updating section to the readme especially since there are a number of versions being updated here and changes required in the helm script files?

File: projects/goharbor/harbor/Makefile [hunk 1]
```diff
@@ -116,6 +117,15 @@ $(OUTPUT_DIR)/$(REPO): $(GIT_PATCH_TARGET)
 
 $(call IMAGE_TARGETS_FOR_NAME, harbor-registry): $(call FULL_FETCH_BINARIES_TARGETS, $(FETCH_BINARIES_TARGETS))
 $(call IMAGE_TARGETS_FOR_NAME, harbor-registryctl): $(call FULL_FETCH_BINARIES_TARGETS, $(FETCH_BINARIES_TARGETS))
+$(call IMAGE_TARGETS_FOR_NAME, harbor-portal): clean-go-cache
```
Comment (jaxesn) [inline projects/goharbor/harbor/Makefile:120]: i am going to move this somewhere else in another PR

**Review by jaxesn — APPROVED** [review]

**Pull request merged**